LoopKitUI/CarbKit/FoodEmojiDataSource.swift

```diff
@@ -97,17 +119,21 @@ private class FoodEmojiDataSource: EmojiDataSource {
         var slow = [
             "🍕", // pizza
             "🥑", // avocado
+            "🥚", // egg
```

Comment:
Source: ChatGPT

> Eggs are considered a slow-release protein source. Slow-release proteins are those that are digested and absorbed slowly by the body, providing a sustained release of amino acids over several hours. Eggs contain high-quality protein, which means they provide all of the essential amino acids needed by the body.
The protein in eggs is primarily found in the egg white, which is made up of approximately 90% water and 10% protein. The protein in egg whites is slow-release, meaning that it is digested and absorbed slowly by the body, providing a sustained release of amino acids over several hours.

Reply:
If anyone has any thoughts on whether egg should be in medium or slow, this is the one I was most unsure about..